### ISSUE TITLE
Cookie banner dismiss "X" is dark gray (A11yyyyyyy!)

### ISSUE THREAD

**Issue by andrewpmiller** [gymnasium/tracker#338]
I think it should be orange... or something more visible.

<img width="449" alt="image" src="https://github.com/gymnasium/tracker/assets/5747506/2ccbebcc-6750-4895-97d4-3839019ce8df">
